src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -375,14 +393,13 @@ private static bool InSymbolHeader(SyntaxNode matchingNode, int position)
 
             if (updatedNode.IsKind(SyntaxKind.ObjectCreationExpression, out ObjectCreationExpressionSyntax? objCreation))
```

Comment:
there is a BaseObjectCreationExpression that is the base type of both tehse creation types.  if there is enough code duplication here, would it be possible to use that instead and unify these blocks?

Reply:
You're right, thanks! Updated.